docs/source/replay.rst

```diff
@@ -33,6 +33,8 @@ Control
 
 - To restart the replay, just press ``Start`` button again.
 
+- The ``debug`` option enables displaying the ID of vehicles, roads and intersections during a mouse hover. **This will cause a slower replaying**, so we suggest using it for debugging purposes.
```

Comment:
```suggestion
- The ``debug`` option enables displaying the ID of vehicles, roads and intersections during a mouse hover. **This will cause a slower replaying**, so we suggest only using it for debugging purposes.
```

Reply:
Please also add this in `index.html`